index.md

```diff
@@ -37,6 +37,34 @@ consumers of your library:
 A scope may be provided to a commit's type, to provide additional contextual information and
 is contained within parenthesis, e.g., `feat(parser): add ability to parse arrays`.
 
+## Examples
+
+### Commit message with description and breaking change in body
```

Comment:
change in footer*

Reply:
Please correct me if I'm wrong, but there is no footer. There is only a body.

Reply:
You are not wrong actually.
The difference is about the content:
The BC can be indicated in [Body](https://conventionalcommits.org/#spec-item-6) or [Footer](https://conventionalcommits.org/#spec-item-7) section.

Sorry this was my bad 😄

Reply:
Is there anything holding this up from being merged at this point?